### PR TITLE
newer version

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: ftp://ftp.freedesktop.org/pub/mesa/mesa-{{ version }}.tar.gz
-  sha256: 9484ad96b4bb6cda5bbf1aef52dfa35183dc21aa6258a2991c245996c2fdaf85
+  sha256: ffd8dfe3337b474a3baa085f0e7ef1a32c7cdc3bed1ad810b2633919a9324840
 
 build:
   skip: True  # [win or py3k]
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: Does this have a python dependency? Why skip py3k?

**Comment by doutriaux1** [comment]
I don't think there's any python at all actually.

**Comment by CJ-Wright** [comment]
You might try removing the python skip statement and then rerendering

**Comment by dougalsutherland** [comment]
@conda-forge-admin, please rerender

**Comment by dougalsutherland** [comment]
Yep, this was caused by the py3k skip. Don't know why that was there, but something about the recent conda-build/conda-smithy upgrade probably changed that from default-interpreted-as-false to default-interpreted-as-true.

**Comment by doutriaux1** [comment]
thanks! Now I can debug this 😉

**Pull request closed without merging**